suites/fs/basic/tasks/cfuse_workunit_suites_truncate_delay.yaml

```diff
@@ -10,7 +10,6 @@ tasks:
 - ceph-fuse:
 - exec:
     client.0:
-      - cd $TESTDIR/mnt.*
-      - dd if=/dev/zero of=./foo count=100
+      - cd $TESTDIR/mnt.* && dd if=/dev/zero of=./foo count=100
       - sleep 2
       - truncate --size 0 ./foo
```

Comment:
Knowing nothing about how the exec task works...

if these don't all get wrapped up into the same shell session, does that mean you have to cd into $TESTDIR/mnt.* again before the truncate command?